jenkins/runlocal

```diff
@@ -1,52 +1,42 @@
 #!/bin/bash -e
-
+#
 # Build the Pbench Server RPM and container, and run functional tests locally.
-# Requires a Fedora, CentOS, or RHEL environment to run.
-
+#
 # NOTE WELL: By default, when the functional tests are run, the infrastructure
 # pod and Pbench Server container are left running by default.  Add the switch,
```

Comment:
One of the "by default"s can go :-), but not necessarily in this PR.

Reply:
I'm going to plead, "I didn't change that or any related line", unless you request some other change.  😁